ts/Dashboards/Plugins/HighchartsComponent.ts

```diff
@@ -604,13 +605,28 @@ class HighchartsComponent extends Component {
             const columnNames = table.modified.getColumnNames();
             const columnAssignment = this.options.columnAssignment ||
                 this.getDefaultColumnAssignment(columnNames);
+            const seriesColumnMap = columnAssignment.seriesColumnMap;
+            // eslint-disable-next-line max-len
+            const pointColumnMap = (seriesColumnMap as HighchartsComponent.seriesColumnMap)?.pointColumnMap;
```

Comment:
Why don't you cast it earlier and remove the redundant casting later.

Reply:
Because the object can be undefined or string (because of using RECORD), so rest parts of code still should verify the type.